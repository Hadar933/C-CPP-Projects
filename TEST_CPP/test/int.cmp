is Empty? 1
size after 100 insertions: 100
values at 27, 89, 18, 12, 88 are 419, 493, 306, 14, 501
size after 200 insertions: 200
values at 132, 94, 197, 71, 91 are 405, 456, 54, 389, 551
size after 300 insertions: 300
values at 281, 233, 261, 109, 170 are 391, 849, 390, 190, 2
size after 400 insertions: 400
values at 344, 21, 337, 231, 146 are 58, 790, 111, 348, 713
size after 490 pop_backs: 9
vector: 
28	179	266	556	618	655	780	726	801	
insert vector 180	2604	909	2583	624	2253	2379	2310	2292	at position 7
output reference to 180
vector: 
28	179	266	556	618	655	780	180	2604	909	2583	624	2253	2379	2310	2292	726	801	
size after inserts: 18
erase many: 655
size after erases: 16
vector: 
28	179	266	655	780	180	2604	909	2583	624	2253	2379	2310	2292	726	801	
insert 46 at 13 ref: 46
vector: 
28	179	266	655	780	180	2604	909	2583	624	2253	2379	2310	46	2292	726	801	
erase from 12 ref: 46
vector: 
28	179	266	655	780	180	2604	909	2583	624	2253	2379	46	2292	726	801	
