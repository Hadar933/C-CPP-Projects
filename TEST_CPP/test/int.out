is Empty? 1
size after 100 insertions: 100
values at 12, 37, 83, 71, 70 are 307, 332, 166, 317, 997
size after 200 insertions: 200
values at 180, 137, 167, 0, 97 are 790, 799, 384, 286, 478
size after 300 insertions: 300
values at 266, 80, 279, 74, 15 are 970, 539, 501, 427, 704
size after 400 insertions: 400
values at 44, 306, 200, 116, 74 are 167, 689, 163, 323, 427
size after 490 pop_backs: 9
vector: 
286	156	347	612	212	775	725	565	689	
insert vector 1503	1989	2970	2613	1767	1872	1887	2484	183	at position 7
output reference to 1503
vector: 
286	156	347	612	212	775	725	943272201	959580473	839462967	154349878	926299953	926429449	942737714	839464760	154417204	80	0	
size after inserts: 18
wrong capC
erase many: 775
size after erases: 16
vector: 
286	156	347	775	725	943272201	959580473	839462967	154349878	926299953	926429449	942737714	839464760	154417204	80	0	
insert 46 at 13 ref: 154417204
vector: 
286	156	347	775	725	943272201	959580473	839462967	154349878	926299953	926429449	942737714	839464760	46	80	0	991	
erase from 12 ref: 839464760
vector: 
286	156	347	775	725	943272201	959580473	839462967	154349878	926299953	926429449	942737714	46	80	0	991	
